### ISSUE TITLE
Create transformer for MEDLINE database records

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss-labs/rialto-etl#72]
If the result is not from MEDLINE it can be skipped.

**Comment by jcoyne** [comment]
Blocked by https://github.com/sul-dlss/rialto/issues/26? or maybe a missing ticket?

**Comment by peetucket** [comment]
MEDLINE records come from the WoS API, but have a different data structure (since they come from a different database on Clarivate's end and Clarivate is not normalizing into a single data structure), so will require a different mapping than WoS database records.  Including them would improve our publication coverage in RIALTO, but can be deferred for if/when we have time to loop back and include them in the publication harvest.
